index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/e98a0f000a.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Osuagwu Lucky James</title>
+  </head>
+  <body>
+    <header class="menu-top">
+      <nav>
+        <ul>
+          <li> <a href="#">LUCKY</a></li>
+          <li><a href="#"><i class="fas fa-bars"></i></a></li>
+        </ul>
+      </nav>
+  </header>
+
+  <section class="about-bio">
+    <div class="introduction">
+      <h1>I’m Lucky<br> Glad to see<br> you!</h1>
+      <p>I’m a software developer! I can help you<br> build a product, feature or website Look<br> through some of my work and<br> experience! If you like what you see and<br> have a project you need coded, don’t<br> hestiate to contact me.</p>
+    </div>
+         <a class="link-up" href="#">LET'S CONNECT</a>
+            <ul class="social-media-icons">
+                <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="www.linkedin.com/in/lucky-james-osuagwu-a8677b171"><i class="fab fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+                <li><a href="https://github.com/Goodluck5956"><i class="fab fa-github"></i></a></li>
+                <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+            </ul>
+    </section>
+
+        <main class="grid">
+          <section class="work-one work">
+            <img src="./templates/SnapshootPortfolio(6).png" alt="snap1">
+            <h2>Tonic</h2>
+            <ul class="work-history">
+              <li class="cover">CANOPY</li>
+                <li><img class="counter" src="./templates/Counter(1).svg" alt="Counter"></li>
+                  <li class="career">Back End Dev</li>
+                <li><img class="counter" src="./templates/Counter(1).svg" alt="Counter"></li>
+              <li class="career">2015</li>
+            </ul>
+            <p class="body-detail">
+              A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.
+            </p>
+            <ul class="courses">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          <form action="#">
+              <button class="btn-body" type="submit" name="submit">see project</button>
+          </form>
+          </section>
```

Comment:
- [ ] Well done on using HTML semantic tags. However, to make your project more semantically correct, I think it's better to wrap all the displayed projects in a `<section class="grid">` and use divs for the cards of the project.
Your webpage will look something like this: 
`<main>`
`<section class="grid">`
`<div class="work-one work">` `</div>`
`<div class="work-one work">` `</div>`
`<div class="work-one work">` `</div>`
`</section>`
`<section class="about-myself"` `</section>`
`</main>`

- [ ] As you can see in the picture the cards have the same width. 
![image](https://user-images.githubusercontent.com/43698511/130152262-1843065a-021a-4755-920b-34287d066047.png)

Your project looks a bit different. Kindly fix the issue.
![image](https://user-images.githubusercontent.com/43698511/130152462-d32fdc88-4742-4c98-af64-875c43851e28.png)